core/http-auth-aws-event-stream/src/main/java/software/amazon/awssdk/http/auth/aws/eventstream/AwsV4EventStreamHttpSigner.java

```diff
@@ -33,6 +33,6 @@ public interface AwsV4EventStreamHttpSigner extends HttpSigner<AwsCredentialsIde
      * @return AwsV4EventStreamHttpSigner
      */
     static AwsV4EventStreamHttpSigner create() {
-        return new DefaultAwsV4EventStreamHttpSigner();
+        return null;
```

Comment:
Hmm, do we intend to return null?

Reply:
Yes, this interface should go away soon.